src/components/image.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FC } from "react"
+import classnames from "classnames"
+
+interface Props {
+  image: string
+  onLoad?: () => void
+  alt?: string
+  imageType?:
+    | "fullWidth"
+    | "fixedCard"
+    | "rounded"
+    | "growingCard"
+    | "fullScreenSlider"
+    | "galleryPicture"
+    | "fullScreenGalleryImage"
+    | "imageInfo"
+    | "maxHeight"
+    | "bigImage"
+  objectFit?: "cover" | "contain"
+}
+
+const Image: FC<Props> = ({ image, objectFit, imageType, onLoad, alt }) => {
+  const cover = classnames({
+    "object-cover": objectFit === "cover",
+    "object-contain": objectFit === "contain",
+  })
+
+  const classes = classnames({
```

Comment:
@lkappeler When you suggested to create an interface you were thinking about something like this right ? 
It looks a bit messy since there's a lot of variations. Would you advice to split it again like I did for objectFit : 1 props for `width`, another one for `height` and then the `classes` ?